src/components/StateBook.jsx

```diff
@@ -1,10 +1,22 @@
 import React from 'react';
-import './Zbook.css';
+// import { useDispatch, useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
+// import { removeBook } from '../redux/BookSlice';
+import './Zbook.css';
 
 function Zbook({
-  ll, l, lll, llll, lllll,
+  ll, l, lll, llll, lllll, Removebook,
 }) {
+  // const books = useSelector((state) => state.books.books);
+  // const dispatch = useDispatch();
+
+  // const Remove = () => {
+  //   dispatch(removeBook({
+  //     Name: 'Dune',
+  //   }));
+  //   console.log(books);
+  // };
+
```

Comment:
- Hi, you have done a nice job. I would advise you not to leave commented code in your files because it makes your work looks dirty and non-professional. please clean your code and ask for a second review 👍